toolkits/global/packages/global-expander/js/expander.js

```diff
@@ -23,8 +24,11 @@ const Expander = class {
 		this._targetTabbableItems = makeArray(target.querySelectorAll(
 			'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
 		));
-		this._isOpen = false;
-
+		if (typeof this._options.TARGET_OPEN_INITIALLY === 'boolean') {
+			this._isOpen = this._options.TARGET_OPEN_INITIALLY;
+		} else {
```

Comment:
perhaps you could remove this else statement and instead set the default value of `TARGET_OPEN_INITIALLY` to be false to save a bit of code?

Reply:
Good point 😅!